submissions/solaryasha/js-dom/js/script.js

```diff
@@ -0,0 +1,98 @@
+        "img/Dota2.jpg"
+    ),
+    new game(
+        "Overwatch",
+        "overwatch",
+        "Overwatch is an online team-based game generally played as a first-person shooter. The game features several different game modes, principally designed around squad-based combat with two opposing teams of six players each. Players select one of over two dozen pre-made hero characters from one of three class types: Damage heroes that deal most of the damage to attack or defend control points, Tank heroes that can absorb a large amount of damage, and Support heroes that provide healing or other buffs for their teammates",
+        "img/overwatch.jpeg"
+    ),
+    new game(
+        "Hearthstone",
+        "hearthstone",
+        "Hearthstone is a free-to-play online digital collectible card game developed and published by Blizzard Entertainment. Originally subtitled Heroes of Warcraft, Hearthstone builds upon the existing lore of the Warcraft series by using the same elements, characters, and relics.The game is a turn-based card game between two opponents, using constructed decks of 30 cards along with a selected hero with a unique power. Players use their limited mana crystals to play abilities or summon minions to attack the opponent, with the goal of destroying the opponent's hero. Winning matches and completing quests earn in-game gold, rewards in the form of new cards, and other in-game prizes.",
+        "img/hearthstone.jpg"
+    ),
+    new game(
+        "Counter-Strike: GO",
+        "counter-strike: global offensive",
+        "Global Offensive, like prior games in the Counter-Strike series, is an objective-based, multiplayer first-person shooter. Two opposing teams, known as the Terrorists and the Counter-Terrorists, compete in game modes to complete objectives, such as securing a location to plant or defuse a bomb and rescuing or guarding hostages.At the end of each round, players are rewarded based on their individual performance with in-game currency to spend on other weapons or utility in subsequent rounds. Winning rounds rewards more money than losing does, and completing objectives such as killing enemies gives cash bonuses. Uncooperative actions, such as killing teammates, results in a penalty",
+        "img/CSGO.jpg"
+    ),
+    
+]
+function gameTemplate (database,element)  {
+    const templateHeader = element.querySelector("h2");
+    templateHeader.textContent = database.header;
+    const templateImage = element.querySelector("img");
+    templateImage.setAttribute("src", database.image);
+    templateImage.setAttribute("alt", database.src);
+    
+    const templateText = element.querySelector("p");
+    templateText.textContent = database.text;
+    
+
+}
+tabs.addEventListener("click", contentGenerator);
+
+function contentGenerator (event) {
```

Comment:
Try to use [Destructuring assignment](https://javascript.info/destructuring-assignment).

Reply:
I have changed the code. Thanks for your suggestion!